CAN-1185

vikasranatalentelgia opened the pull request:
Manage Support: clicking on camp name should take user to topic detail page

vikasranatalentelgia committed:
CAN-1185

Manage Support: clicking on camp name should take user to topic detail page

vikasranatalentelgia committed:
Build issue resolved

Build issue resolved

akuks commented on src/components/ComponentPages/ManageSupport/index.tsx:
```diff
@@ -53,6 +53,8 @@ const ManageSupport = () => {
     const body = { topic_num: topicNum };
 
     let res = await getAllUsedNickNames(topicNum && body);
+    console.log(res, "pop");
```
please remove console.log

vikasranatalentelgia replied:
Done

akuks commented on src/components/ComponentPages/UserProfile/_test_/userProfile.test.tsx:
```diff
@@ -33,20 +33,30 @@ const noData = false;
+        userSupportedCampsList={userSupportedCampsList}
+      />
+    );
+    expect(screen.queryByTestId(labels.name)).toBeNull();
```
Can we create an array and then iterate?

vikasranatalentelgia replied:
Need to check if it can be done in test cases.

vikasranatalentelgia committed:
Merge branch 'development' into Fixes

vikasranatalentelgia committed:
CAN-1174,CAN-1175

Getting "Something went wrong" error on supported camps page
Issues when user click on Cancel button from supported camps page

vikasranatalentelgia committed:
CAN-1138

Delegated supported camps are listing like direct supported camps on user supports page
